src/web/components/MostViewedList/MostViewedListItem.tsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import { css } from 'emotion';
+
+import { palette } from '@guardian/src-foundations';
+import { headline } from '@guardian/src-foundations/typography';
+import { QuoteIcon } from '@root/src/web/components/QuoteIcon';
+import { AgeWarning } from '@root/src/web/components/AgeWarning';
+
+import { TrailType } from './MostViewedList';
+
+const listItemStyles = css`
+    list-style: none;
+    padding-top: 4px;
+    margin-bottom: 12px;
+    border-top: 1px solid ${palette.neutral[86]};
+
+    &:first-child {
+        padding-top: 0;
+        border-top: none;
+    }
+`;
+
+const linkTagStyles = css`
+    text-decoration: none;
+    font-weight: 500;
+    ${headline.tiny()};
+
+    &:link,
+    &:active {
+        color: ${palette.neutral[7]};
+    }
+
+    &:hover .linkText {
+        text-decoration: underline;
+    }
+
+    &:visited .linkText {
+        color: ${palette.neutral[46]};
+    }
+`;
+
+const textWrapperStyles = css`
+    display: flex;
+`;
+
+const imageWrapperStyles = css`
+    width: 72px;
+    min-height: 72px;
+    padding-top: 3px;
+    margin-right: 10px;
+    overflow: hidden;
+    position: relative;
+    box-sizing: content-box;
+    flex-grow: 0;
+    flex-shrink: 0;
+`;
+
+const imageTagStyles = css`
+    position: absolute;
+    width: auto;
+    height: 72px;
+    left: -24px;
+    clip-path: circle(36% at 50% 50%);
+`;
+
+const ageWarningStyles = css`
+    margin-top: 8px;
+`;
+
+const liveKicker = (colour: string) => css`
+    color: ${colour};
+    font-weight: 700;
+
+    &::after {
+        content: '';
+        display: inline-block;
+        font-weight: 900;
+        margin: 0 2px;
+    }
+`;
+
+function getColour(pillar: Pillar) {
+    return palette[pillar].main;
+}
+
+type Props = {
+    trail: TrailType;
+    position: number;
+};
+
+export const MostViewedListItem = ({ trail, position }: Props) => (
+    <li className={listItemStyles}>
+        <a
+            className={linkTagStyles}
+            href={trail.url}
+            data-link-name={'article'}
+        >
+            <div className={textWrapperStyles}>
+                <div className={imageWrapperStyles}>
+                    <img
+                        src={trail.image}
+                        role="presentation"
```

Comment:
I don't think we need the `presentation` role here, when we have the empty `alt`. Ideally, I'd have thought the alt text would be useful but, for now, I think we should just remove the role?

This is a wider spread issue but it should be an *empty* alt not an attribute without a value, which may be a difference between `alt=""` and `alt={""}`.

Reply:
The linter won't accept an empty string for the `alt` attribute unless the element has a `role` set to `presentation`. I couldn't find any way around this unless we provide some text value to `alt` but we don't have anything that's descriptive of the image.

In this case it might actually be better to use the `role=presentation` to convey the image has no text description and should be treated as site furniture.

Thoughts? @gtrufitt

Reply:
Ah ok, yeah, probably no major *harm* in `role=presentation`, more that it's probably unnecessary so, not sure why the linter is crying about it. 

Fine to leave as is, I'll card up seeing about the actual `alt` attribute being populated, which would require the image's alt to be passed through.